CHANGELOG

```diff
@@ -10,6 +10,8 @@ Qtile 0.x.x, released xxxx-xx-xx:
         - `libqtile.window` has been moved to `libqtile.backend.x11.window`; a migration has been added for this.
     !!! deprecation warning !!!
         - 'main' config functions, deprecated in 0.16.1, will no longer be executed.
+    !!! Notice for packagers - new dependencies !!!
+        - Tests now require the 'dbus-next' python module plus 'dbus-launch' and 'notify-send' applications
```

Comment:
Is it worth leaving this in if we've now got ability to skip tests where they're not installed?

Reply:
Maybe just add it to the `tox.init` test dependencies? I think it's all then automatic when using tox anyway.

Reply:
`dbus-next` is in the tox dependencies. My concern was more around the `dbus-launch` and `notify-send` apps. As you said in an earlier comment, these apps are packaged different in different distros. 

For example, on Arch, the PKGBUILD file for qtile runs the test suite in its `check` function (see https://github.com/archlinux/svntogit-community/blob/packages/qtile/trunk/PKGBUILD). There are two options for the Arch packagers: 1) add the relevant dependencies to `checkdepends`; or 2) do nothing and the test will now be skipped if those requirements aren't installed.

Option 2 doesn't seem too bad.

Reply:
Ah I see ok. In that case it's probs best to keep this changelog entry so they can update checkdepends (if they're reading it) and no issues if they do not